src/main/kotlin/org/jitsi/jibri/selenium/JibriSelenium.kt

```diff
@@ -259,10 +272,17 @@ class JibriSelenium(
             try {
                 HomePage(chromeDriver).visit(callUrlInfo.baseUrl)
 
+                var callStatsUsername = "jibri"
+                if (jibriSeleniumOptions.callStatsUsername.isNotEmpty()) {
+                    callStatsUsername = jibriSeleniumOptions.callStatsUsername
+                } else if (MainConfig.jibriId.isNotEmpty()) {
+                    callStatsUsername = MainConfig.jibriId
+                }
```

Comment:
You lost the case where it defaults to `"jibri"` if the `jibriId` is empty.

Reply:
This case is covered by initializing callStatsUsername = "jibri" at the beginning, and only overriding it with jibriId if callStatsUsername is empty and if jibriId is not empty

Reply:
🤦  Missed that. 👍